src/traces/sankey/base_plot.js

```diff
@@ -34,3 +40,94 @@ exports.clean = function(newFullData, newFullLayout, oldFullData, oldFullLayout)
         oldFullLayout._paperdiv.selectAll('.sankey').remove();
     }
 };
+
+exports.updateFx = function(gd) {
+    for(var i = 0; i < gd._fullData.length; i++) {
+        subplotUpdateFx(gd, i);
+    }
+};
+
+var dragOptions = [];
```

Comment:
Why do we need to track the `dragOptions` entries in the file scope?

Reply:
In the current implementation we don't. 

I :hocho:'ed it in 20d8c92d8454cc804e71b84a90469df807d14898